source/views/contacts/contact-detail.js

```diff
@@ -68,22 +73,35 @@ export class ContactsDetailView extends React.PureComponent {
 
   render() {
     const contact = this.props.navigation.state.params.contact
+    const headerImage = contact.image &&
+      contactImages.hasOwnProperty(contact.image)
+      ? contactImages[contact.image]
+      : null
     return (
-      <Container>
-        <Title selectable={true}>{contact.title}</Title>
+      <ScrollView>
```

Comment:
Why did this need the extra outer view instead of just adding the image above the title?

Reply:
Because the inner view was a glamorous style component that applied a margin on its children. If I add the image above the title, the image will not expand to the width and height of the view.